Readability.js

```diff
@@ -1058,7 +1067,6 @@ Readability.prototype = {
   _findNextPageLink: function(elem) {
     var uri = this._uri;
     var possiblePages = {};
-    var allLinks = elem.getElementsByTagName('a');
```

Comment:
Err, this is used a few lines down... I mean, the code isn't currently being called at all, but we should try not to break it like this. :-)